crates/wasmtime/src/func.rs

```diff
@@ -722,7 +727,10 @@ impl Func {
         (get15, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15)
     }
 
-    pub(crate) fn store(&self) -> &Store {
+    // This should be `pub(crate)` but the C API happens to need it in one
+    // place.
+    #[doc(hidden)]
+    pub fn store(&self) -> &Store {
```

Comment:
Is this a place in the C API where the store should be passed in?

Reply:
This seems like a fine API to expose though, especially because `Caller` below exposes it too

Reply:
This is needed for the `wasi_instance_bind_import` WASI C API function -- do you happen to know if this is part of the C API or part of wasi?

Reply:
Ah I think that's just part of WASI, but this seems fine to add anyway